src/SFA.DAS.Payments.MatchedLearner.Database/dbo/Tables/dbo.Period.sql

```diff
@@ -0,0 +1,26 @@
+﻿CREATE TABLE [dbo].[Period]
+(
+	[Id] BIGINT NOT NULL PRIMARY KEY,
+    [PriceEpisodeId] BIGINT NOT NULL CONSTRAINT FK__Period_PriceEpisode FOREIGN KEY REFERENCES [dbo].[PriceEpisode] (Id),
+    [Period] INT NOT NULL, 
+    [IsPayable] BIT NOT NULL, 
+    [AccountId] BIGINT NULL, 
+    [ApprenticeshipId] BIGINT NULL, 
+    [ApprenticeshipEmployerType] INT NULL, 
+    [TransferSenderAccountId] BIGINT NOT NULL, 
+    [FailedDataLock1] BIT NOT NULL,
```

Comment:
add default constrain to false on all these columns

Reply:
fixed